Multisig/Data/Services/Utils/String+Utils.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  String.swift
+//  Multisig
+//
+//  Created by Moaaz on 7/19/21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+
+extension String {
+    func compareNumeric(_ value: String) -> ComparisonResult {
+        return compare(value, options: .numeric)
```

Comment:
I think we already have imported the "Version" swift package, I think this should be used in order to compare versions.